api/billingd/service/service.go

```diff
@@ -512,19 +515,27 @@ func periodToPb(period Period) *pb.Period {
 	}
 }
 
-func (s *Service) usageToPb(usage map[string]Usage) map[string]*pb.Usage {
+func (s *Service) dailyUsageToPb(usage map[string]Usage) map[string]*pb.Usage {
+	start, end := getCurrentDayBounds()
 	res := make(map[string]*pb.Usage)
 	for k, u := range usage {
 		product, ok := s.products[k]
 		if ok {
-			res[k] = getUsage(product, u.Total)
+			res[k] = getUsage(product, u.Total, Period{UnixStart: start, UnixEnd: end})
 		}
 	}
 	return res
 }
 
-func getUsage(product Product, total int64) *pb.Usage {
-	freeUnits, paidUnits := getDailyUnits(product, total)
+func getCurrentDayBounds() (int64, int64) {
+	now := time.Now()
+	start := time.Date(now.Year(), now.Month(), now.Day(), 0, 0, 0, 0, time.Local)
+	end := time.Date(now.Year(), now.Month(), now.Day(), 24, 0, 0, 0, time.Local)
```

Comment:
Technically this should be minus one second, but it's only for display purposes where it's nice to show just the year, month, and day.